src/cli.rs

```diff
@@ -6,12 +6,15 @@ use std::ffi::OsString;
 pub struct TrinConfig {
     pub web3_transport: String,
     pub web3_ipc_path: String,
-    pub web3_http_port: u32,
+    pub web3_http_port: u16,
```

Comment:
Yeah, this is basically like a hangover coming from python land when I'm used to treating all integers as the same type (`u32`) - need to drink some coffee and sober up here in rust world.